sdk/vaa/structs.go

```diff
@@ -120,6 +120,10 @@ func (c ChainID) String() string {
 		return "algorand"
 	case ChainIDNear:
 		return "near"
+	case ChainIDAptos:
```

Comment:
These should be added to structs_tests.go as well.

Reply:
addressed